src/main/java/lotto/enums/RewardEnum.java

```diff
@@ -0,0 +1,35 @@
+package lotto.enums;
+
+public enum RewardEnum {
+
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
큰 숫자는 `_`를 활용해 가독성을 향상 시킬 수 있습니다
```suggestion
    SIX(6, 2_000_000_000);
```